### DISCUSSION TITLE
Deepin 20.6 installation error messages in virt-manager

### DISCUSSION THREAD

**Discussion by serial02** [linuxdeepin/developer-center#3212] [Features Request & Ideas | 特性请求 & 头脑风暴]
Hello everybody

 When installing Deepin 20.6 on virt-manager we can view the installation log and see that there are some errors in the Distro installation and my doubt is as follows:

 1- Are these errors sent to the developer team?
 2- If yes, why are users not warned at the time of installation?
 3- If not, is there a possibility for the team of developers and technical engineers to put the installation log on the desktop to be sent later by users, if they wish?
 4- If it is not possible to put this installation log on the desktop to be sent later, can you ask for user consent to automatically send this information?
 5- Wouldn't this log help the technical team to find several faults at the time of installation and possibly be corrected in future installations?

 If you need the image with the error messages at the time of installation, I will send them later.

 Thank you in advance for your attention

**Comment by BLumia** [comment]
> Are these errors sent to the developer team?

No.

> If not, is there a possibility for the team of developers and technical engineers to put the installation log on the desktop to be sent later by users, if they wish?

Yes. The installer provided by deepin will prompt a QR code and related message that can be seen by user if anything wrong happen during installation.

**Reply by serial02** [reply]
> Yes. The installer provided by deepin will prompt a QR code and related message that can be seen by user if anything wrong happen during installation.

And if it is not seen by the user which is the most common to occur?  Who migrates from Windows is always next and next(!) Many don't even look at what happened if something doesn't "scream"
 Couldn't the installation log automatically track the installation by capturing the word "installation error" or just "error" and do all this automatically, asking the user to send it to the base for further review of what is wrong?

**Reply by BLumia** [reply]
> And if it is not seen by the user which is the most common to occur?

No, if there is something wrong, you cannot proceed anymore and then the screen that indicates how to report the issue will be shown.

> Couldn't the installation log automatically track the installation by capturing the word "installation error" or just "error" and do all this automatically, asking the user to send it to the base for further review of what is wrong?

When the installer gets any serious error, the user WILL see the troubleshooting screen that does exactly what you ask for. If you didn't see that screen, it means your setup is okay and there is no other thing you need to do. If the installation process is done and you got any issue after that process, it means there are some other things going wrong, and that will no longer be related to the installation process (or it's a bug).
